src/main/resources/mybatis/groundwaterDailyValueWaterML.xml

```diff
@@ -33,7 +52,9 @@
                           observered_property_name,
                           observered_property_reference,
                           statistic,
-                          statistic_reference
```

Comment:
This comma diff issue is always annoying. I wonder if there is some convention we could follow to reduce it. BTW, I do not like leading commas either but I could get over it.